pkg/cloud/cloud.go

```diff
@@ -25,21 +26,30 @@ import (
 	"github.com/aws/aws-sdk-go/service/fsx"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/klog/v2"
+	"sigs.k8s.io/aws-fsx-openzfs-csi-driver/pkg/util"
 	"strings"
 	"time"
 )
 
+const (
+	// DefaultFileSystemSize represents the default size used
+	// this is the minimum FSx OpenZFS size
+	DefaultFileSystemSize int64 = 64
+	DefaultFileSystemType       = "OPENZFS"
+)
+
 // Polling
 const (
-	// SnapshotPollCheckInterval specifies the interval to check if the resource is ready;
-	// this needs to be shorter than the csi snapshotter timeout.
-	SnapshotPollCheckInterval = 15 * time.Second
+	// PollCheckInterval specifies the interval to check if the resource is ready;
+	// this needs to be shorter than the timeout.
+	PollCheckInterval = 15 * time.Second
```

Comment:
I don't have a strong opinion on combining the Snapshot PollCheckInterval and Dynamic Provisioning PollCheckInterval, but I'd like to understand why for Dynamic Provisioning it's 30 seconds in the lustre repo and if lowering it here will cause user impact

Reply:
I'm not quite familiar with the time it takes to create a Lustre filesystem. If it takes a while I'd assume it was to decrease the number times the controller pings FSx's API. The only user impact I see by lowering the value is an increase in network traffic with the added benefit of execution time.

Personally I'm fine with keeping the interval consistent between operations since there shouldn't be any drastic time differences between the time it takes CreateVolume, CreateFilesystem and CreateSnapshot to become available. If we want to separate intervals based on operations I'm also fine with that.